english/check-your-phone/check-your-phone.md

```diff
@@ -0,0 +1,12 @@
+Pull out your phone to check the date and see where you are.
+You remember hearing somewhere that you can't read in a dream.
+As you look at your phone, you realize that you can't make
+out what it says...
+
+Do you:
+
+[Decide this is just a dream and buy a t-shirt from CustomShirts.com] (http://CustomShirts.com)
+
+[Realize that it's just a dream](https://github.com/danielstanojevic/create-your-own-adventure/blob/master/english/dream/dream.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.